vividus-extension-selenium/src/main/java/org/vividus/steps/ui/GenericWaitSteps.java

```diff
@@ -132,6 +137,47 @@ public String toString()
         });
     }
 
+    /**
+     * Waits for an <b><i>element</i></b> with the specified <b>locator</b> to stop moving
+     * @param locator locator to locate element
+     */
+    @When("I wait until element located by `$locator` stops moving")
+    public void waitForElementStopsMoving(Locator locator)
+    {
+        uiContext.getOptionalSearchContext().ifPresent(context -> {
+            locator.getSearchParameters().setWaitForElement(false);
+            IExpectedSearchContextCondition<Boolean> condition = new IExpectedSearchContextCondition<>() {
+                private final List<Point> points = new ArrayList<>();
+
+                @Override
+                public Boolean apply(SearchContext searchContext)
+                {
+                    try
+                    {
+                        return searchActions.findElement(context, locator).map(WebElement::getLocation).map(s ->
+                        {
+                            boolean samePosition = !points.isEmpty() && s.equals(points.get(points.size() - 1));
+                            points.add(s);
+                            return samePosition;
+                        }).orElse(false);
+                    }
+                    catch (StaleElementReferenceException e)
+                    {
+                        return false;
+                    }
+                }
+
+                @Override
+                public String toString()
+                {
+                    return String.format("element located by \"%s\" stops moving. Detected locations: %s",
+                            locator.toHumanReadableString(), points);
```

Comment:
how does assertion message look like?

Reply:
<img width="691" alt="image" src="https://github.com/vividus-framework/vividus/assets/18533593/a3799f16-9e1d-4258-ad75-2f724ea2186a">

Reply:
what is the new assertion error message?